pyuvdata/uvcal.py

```diff
@@ -432,6 +432,28 @@ def select(self, antenna_nums=None, antenna_names=None,
         if run_check:
             self.check(run_check_acceptability=run_check_acceptability)
 
+    def convert_to_gain(self):
+        if self.cal_type == 'gain':
+            raise ValueError('The data is already a gain cal_type.')
+        elif self.cal_type == 'delay':
+            phase_array = np.zeros((self.Nants_data, self.Nspws, self.Nfreqs, self.Ntimes, self.Njones))
+            for si in range(self.Nspws):
+                temp = -2 * np.pi * np.dot(self.delay_array[:, si, :, :, np.newaxis],
+                                           self.freq_array[si, np.newaxis, :])
+                temp = np.transpose(temp, (0, 3, 1, 2))
+                phase_array[:, si, :, :, :] = temp
+
+            amplitude_array = np.ones_like(phase_array)
```

Comment:
It probably is. I guess I was trying to be explicit, but I can take that out and note it in a comment.